--- setup.py	2018-08-14 14:22:51.741198727 -0400
+++ setup.py.new	2018-08-14 14:25:04.469391280 -0400
@@ -48,7 +48,7 @@
 
 VERSION = hmmlearn.__version__
 
-install_requires = ["sklearn>=0.16"]
+install_requires = ["scikit-learn>=0.16"]
 tests_require = install_requires + ["pytest"]
 docs_require = install_requires + [
     "Sphinx", "sphinx-gallery", "numpydoc", "Pillow", "matplotlib"

--- hmmlearn.egg-info/requires.txt	2018-08-14 14:23:18.341237317 -0400
+++ hmmlearn.egg-info/requires.txt.new	2018-08-14 10:21:51.756708103 -0400
@@ -1,6 +1,6 @@
 
 [docs]
-sklearn>=0.16
+scikit-learn>=0.16
 Sphinx
 sphinx-gallery
 numpydoc
@@ -8,5 +8,5 @@
 matplotlib
 
 [tests]
-sklearn>=0.16
+scikit-learn>=0.16
 pytest
